admin/registration_providers/views.py

```diff
@@ -356,3 +358,40 @@ def post(self, request, *args, **kwargs):
             }
         # Return a JsonResponse with the JSON error or the validation error if it's not doing an actual migration
         return JsonResponse(response_data)
+
+
+class ShareSourceRegistrationProvider(PermissionRequiredMixin, View):
+    permission_required = 'osf.change_registrationprovider'
+    raise_exception = True
+
+    def get(self, request, *args, **kwargs):
+        provider = RegistrationProvider.objects.get(id=self.kwargs['registration_provider_id'])
+
+        resp = requests.post(
+            f'{website_settings.SHARE_URL}api/v2/sources/',
+            json={
+                'data': {
+                    'type': 'Source',
+                    'attributes': {
+                        'homePage': provider.domain if provider.domain else f'{website_settings.DOMAIN}/registration/{provider._id}/',
```

Comment:
i think this is the wrong url for a registration provider
```suggestion
                        'homePage': provider.domain if provider.domain else f'{website_settings.DOMAIN}/registries/{provider._id}/',
```